pydantic/main.py

```diff
@@ -102,6 +102,20 @@ def __call__(self, schema: Dict[str, Any], model_class: Type['Model']) -> None:
     except AttributeError:
         compiled = False
 
+
+_T = TypeVar('_T')
+
+
+def __dataclass_transform__(
+    *,
+    eq_default: bool = True,
+    order_default: bool = False,
+    kw_only_default: bool = False,
```

Comment:
humm, shouldn't this be `True`?

Reply:
This just copies the signature of `dataclass_transform` as it would appear in the `typing` module.  It's set to `True` for Pydantic when it's called in the decorator.

Reply:
🙈 sorry.